src/plugins/validate-semantic/validators/oas3/refs.js

```diff
@@ -1,3 +1,4 @@
+
 export const validateOAS3RefsForRequestBodiesReferenceRequestBodyPositions = () => sys => {
```

Comment:
let's drop this leading newline!

```suggestion
export const validateOAS3RefsForRequestBodiesReferenceRequestBodyPositions = () => sys => {
```

Reply:
Is this the same as the below comment?

Reply:
it's not! there's a new blank line at the top of this file, and I'd like to remove that 🙂

Reply:
Done! I didn't see it. My bad.